jupyterlab_gitsync/jupyterlab_gitsync/tests/nbinit_helper_test.py

```diff
@@ -0,0 +1,90 @@
+import unittest
+import subprocess
+from jupyterlab_gitsync.nb_handlers import NotebookInitHandler
+
+def make_setup():
```

Comment:
Use the setUp() method instead: https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp